### PR TITLE
Add documentation for HTTP API

### PR DESCRIPTION
**Pull request by sideninja**
## Description
This PR adds documentation for AN HTTP API leveraging redoc technology.
______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

**Comment by 10thfloor** [comment]
The page can now be viewed at `/http-api`

File: docs/gatsby-config.js [hunk 1]
```diff
@@ -284,6 +285,7 @@ const sections = [
       Reference: [
         "[Available Flow SDKs](/sdks/)",
         "[Flow Access API](/access-api/)",
+        "[Flow HTTP API](/http/an/)]",
```
Comment (10thfloor) [inline docs/gatsby-config.js:288]:
```suggestion
        "[Flow HTTP API](/http-api)]",
```

**Comment by kerrywei** [comment]
Screen Shot 2022-02-08 at 11 29 40 AM
@10thfloor is my URL correct?

**Comment by 10thfloor** [comment]
@kerrywei @srinjoyc @sideninja 
Ok this is verified working now. You can verify using the preview URL. 

I'm having an issue with the sidebar links, but that shouldn't block this form going out. 
I added a feature on the homepage, that we can use for now. It would be great to have a blog post or something to link to from the feature as well?

Once I fix the sidebar issue (menu refuses to render a link to the new page for some reason) I'll add a link under 'references'

**Comment by psiemens** [comment]
Great idea to use the redoc React component!

> The search bar is not clickable and I'm having an issue with the sidebar links, but that shouldn't block this from going out.

Hmm, I also can't click any of the links inside the Redoc component and it makes it pretty hard to use. I think we should try to fix that before releasing.

File: docs/content/index.mdx [hunk 2]
```diff
@@ -16,8 +16,43 @@ import { IconGithub, IconDiscord, IconDiscourse } from "gatsby-theme-flow/src/ui
...
+</TrackingLink>
+ {" "}
+ and {" "}
+<TrackingLink href="https://github.com/onflow/fcl-js/blob/master/packages/sdk/CHANGELOG.md#0057-alpha1----2022-01-21
+" eventName="Homepage_link_/fcl/_clicked">
+  FCL (Flow Client Library)
+</TrackingLink>
+today!
+    <p>
+      Oh, thats new.
```
Comment (kerrywei) [inline docs/content/index.mdx:49]: is this intended?

**Review by kerrywei — APPROVED** [review]
no other concerns, except we want to provide the right URL based on environment, per our offline sync

File: docs/content/index.mdx (hunk 2 above)
Comment (10thfloor) [reply docs/content/index.mdx:49]: Yes :) We can edit later.

**Pull request merged**